src/protocols/device_path.rs

```diff
@@ -64,3 +64,19 @@ pub struct HardDriveMedia {
     pub partition_format: u8,
     pub signature_type: u8,
 }
+
+pub struct Media {
+    pub header: Protocol,
+}
+
+impl Media {
+    pub const SUBTYPE_HARD_DRIVE: u8 = 0x01;
```

Comment:
EDK2 uses `MEDIA_HARDDRIVE_DP`, so should be use `SUBTYPE_HARDDRIVE` as well? The spec has no clear identifier for this, but EDK2 uses `HARDDRIVE` consistently throughout the code-base.

Reply:
Amended as requested in latest push.